### ISSUE TITLE
HTTP Error 500.19 - Internal Server Error on /api/latest

### ISSUE THREAD

**Issue by KShoukry** [codewaregames/UnrealGameSync#1]
<img width="1635" alt="image" src="https://user-images.githubusercontent.com/22622413/164297542-8371f81d-a9ff-4dd4-b5c5-c506ac69bf8d.png">

I followed the instructions in this repo and this is what happens when I go to /api/latest. I'm happy to provide more information but I am not sure which information is relevant.

**Comment by SloppyShovel** [comment]
The set Unreal Game Sync up in IIS it's a bit more complicated because IIS don't really support .Net Core, i also don't recommend using this anymore has Epic has rewritten UnrealGameSync to .Net Core.

**Comment by KShoukry** [comment]
> The set Unreal Game Sync up in IIS it's a bit more complicated because IIS don't really support .Net Core, i also don't recommend using this anymore has Epic has rewritten UnrealGameSync to .Net Core.

Thank you for your reply. I was following the instructions from this repo which recommend IIS:
https://github.com/codewaregames/UnrealGameSync/wiki/MetadataServer

I'll try the "not recommended" method and see if I can get a better result.

**Comment by SloppyShovel** [comment]
Yes it is recommend, but it's not recommended to use this repo anymore, the UnrealGameSync has been updated you should use the one from Epic Games Unreal Engine Repo, this is outdated and won't be updated anymore.

**Comment by KShoukry** [comment]
Ah okay. I will pull the code form Epic Games. This does work now, but even if I make the app start on startup, that doesn't really work for a cloud VM cause no user will be logging into it. I'll need a way for it to be started on computer start up like a windows service. Is there any recommendation you have or standard practice?
